Emby.Server.Implementations/Data/BaseSqliteRepository.cs

```diff
@@ -224,6 +205,25 @@ protected virtual void Dispose(bool dispose)
                 return;
             }
 
+            if (dispose)
+            {
+                WriteLock.Wait();
+                try
+                {
+                    WriteConnection.Dispose();
```

Comment:
shouldn't you do `WriteConnection = null` here as well?

Reply:
No, the field is readonly, can't do that and isn't really needed.